book/07-git-tools/sections/submodules.asc

```diff
@@ -245,10 +245,10 @@ Submodule DbConnector c3f01dc..d0354fc:
   > better connection routine
 ----
 
-If you commit at this point then you will lock the submodule into having the new code when other people update.
+Sollten Sie zu diesem Zeitpunkt einen Commit ausführen, dann wird das Submodul so gesperrt, dass es den neuen Code erhält obwohl andere Personen Aktualisierungen vornehmen.
```

Comment:
Ist das so technisch korrekt?

Reply:
Geben wir dem Ganzen mal Namen: ich habe Repository`myproject`, und darin `gtest` als submodul. Ich sehe dann in einem kleinen, bei mir durchgeführten Beispiel folgendes Verhalten: ich aktualisiere (fetch+merge) neue Commits im Submodul `gtest` von `myproject` und committe das in `myproject`. Wenn nun ein Klon von `myproject` diesen Commit pullt, sieht er mit `git status` dass sein submodul `gtest` nicht mehr aktuell ist und der Klon das Submodul updaten sollte. Ich schlage also folgende Übersetzung vor:
```suggestion
Sollten Sie zu diesem Zeitpunkt einen Commit ausführen, dann wird für das Submodul festgelegt, dass es in allen Repositories den neuen Code enthalten soll. Das bedeutet, wenn andere Personen in Klonen Ihres Repositories `git submodule update` ausführen, bekommen sie auch den neuen Code des Submoduls.
```
Das ist etwas länger, weil ich finde, dass die englische Version hier zu kurz ist.